.github/workflows/js-client-update-docs.yml

```diff
@@ -0,0 +1,65 @@
+name: Documentation Update
```

Comment:
Not sure, I copied all the workflows from the sddk and aadapted the routes where neceessary. I'm not sure if it will be on the template because it is not absolutelly necesary for an external developer, but all our plugins are going to need it

Reply:
Maybe we could leave these lines but commented out on the template?